spec/pane-spec.coffee

```diff
@@ -375,6 +377,27 @@ describe "Pane", ->
       pane.destroyInactiveItems()
       expect(pane.getItems()).toEqual [item2]
 
+  describe "::showSaveDialog()", ->
+    pane = null
+    item =  null
+
+    beforeEach ->
+      item = new Item("A")
+      pane = new Pane(items: [item])
+      atom.showSaveDialogSync = jasmine.createSpy("item.showSaveDialog");
+
+    describe "item implements ::showSaveDialog", ->
+      it "calls item::showSaveDialog", ->
```

Comment:
`"calls showSaveDialog and saves the item with the results of the returned promise"`